submissions/sprint_b/app.py

```diff
@@ -0,0 +1,21 @@
+from . import app
+from sprint_b.helpers import EpithetGenerator, Vocabulary, FileManager
+
+@app.route('/')
+def generate_epithet():
+    return EpithetGenerator.get_rand("../../resources/data.json")
+
+@app.route('/vocabulary')
+def vocabulary():
+    columns = Vocabulary.from_json("../../resources/data.json")
+    column1 = "Column 1: {}".format(columns[0]["Column 1"])
+    column2 = "Column 2: {}".format(columns[0]["Column 2"])
+    column3 = "Column 3: {}".format(columns[0]["Column 3"])
+    return "{}\n\n{}\n\n{}".format(column1, column2, column3)
+
+@app.route('/amount/<int:num>')
+def multiple_epithets(num):
+    epithets = ""
+    for n in range(num):
```

Comment:
Your generate_epithet method should accept the quantity of epithets to generate and return a list of epithets. All of the logic for looping some quantity of times should be part of your generate_epithet method.

Define an EpithetGenerator class with methods to:
select one random word from each column of the list.
generate a quantity of epithets from a vocabulary file loaded from a path.